SSLP_analyzer/SSLP_analyzer/settings.py

```diff
@@ -0,0 +1,130 @@
+"""
+Django settings for SSLP_analyzer project.
+
+Generated by 'django-admin startproject' using Django 4.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/4.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/4.2/ref/settings/
+"""
+
+from pathlib import Path
+import os
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve().parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'django-insecure-)i*d(d9v=xll=grvvw6h+(d(6_-pn)h+g_tx6%t*s&=xj5(utu'
```

Comment:
This should not be here...

Reply:
This is fixed in #11 where it fetches the key from an enviroment variable which is saved in a .env file.